kind_with_registry.sh

```diff
@@ -6,6 +6,9 @@ set -o errexit
 
 go install sigs.k8s.io/kind@v0.15.0
 
+[[ $(type -P kind) ]] || ( echo "kind is not in PATH" ;  exit 2 )
```

Comment:
Hmm looks like the github action defaults to `sh` please add `#!/bin/bash` shebang

Reply:
fixed